.github/workflows/build-wsl.yaml

```diff
@@ -134,6 +135,31 @@ jobs:
           $pwd = ConvertTo-SecureString  '${{ secrets.CERTIFICATE_PASSWORD }}' -AsPlainText -Force
           Import-PfxCertificate -Password $pwd -CertStoreLocation Cert:LocalMachine\Trust -FilePath certificate\certificate.pfx
           Import-PfxCertificate -Password $pwd -CertStoreLocation Cert:CurrentUser\My -FilePath certificate\certificate.pfx
+      - name: Check if packaging the splash app is necessary
+        working-directory: ${{ env.workDir }}
+        shell: bash
+        run: |
+          BUILD_TREE_HASH=$(sha256sum DistroLauncher-Appx/Directory.build.props | cut -f1 -d' ')
+          META_TREE_HASH=$(sha256sum meta/Ubuntu18.04LTS/generated/DistroLauncher-Appx/Directory.build.props | cut -f1 -d' ')
+          if [ ${BUILD_TREE_HASH} != ${META_TREE_HASH} ]; then
+            echo "withFlutterSplash=true" >> $GITHUB_ENV
+          else
+            echo "Skipping Flutter steps due hashes comparing equal:"
```

Comment:
The message could be enhanced to be due to the result: "Directory.build.props not doing anything" or so.